src/sidebar/barrel/util/barrelUtil.ts

```diff
@@ -35,6 +37,8 @@ function saveBarrel(barrel: Barrel) {
 function removeBarrel(barrel: Barrel) {
     const barrels = getSavedBarrels().filter(savedBarrel => savedBarrel.id !== barrel.id);
     saveBarrels(barrels);
+    ScreensStoreProvider.clearCacheFor(barrel.id);
+    BarrelDetailsStoreProvider.clearCacheFor(barrel.id);
```

Comment:
Wouldn't it be better if you check barrel type and then decide which Provider's `clearCacheFor` method to call? Or is there any possibility that both providers would be taken into account?

Reply:
for styleguides it's both

Reply:
Gotcha! Then LGTM 👍